akka-remote/src/main/java/akka/remote/artery/compress/TopHeavyHittersScanning.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote.artery.compress;
+
+public class TopHeavyHittersScanning {
```

Comment:
yeah, that's what I'm saying while + force its termination via putting a high index.
Anyway, will rewrite to pure scala :)